jobs/delete_unsaved_candidates.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import datetime
+from skyportal.models import init_db, Source, DBSession
+from baselayer.app.env import load_env
+
+
+env, cfg = load_env()
+init_db(**cfg["database"])
+
+cutoff_datetime = datetime.datetime.now() - datetime.timedelta(
+    days=cfg["misc.days_to_keep_unsaved_candidates"]
```

Comment:
perhaps we should check to make sure this config variable is integer and between 1 and X days.

Reply:
Sure, what's a reasonable max? 30 days?

Reply:
Done